src/test/createQuestionnaire.test.js

```diff
@@ -0,0 +1,151 @@
+const {createQuestionnaire} = require('../controller/createQuestionnaireController');
+import Questionnaire from '../models/dream';
+
+describe('Creation of a new questionnaire', () => {
+  test('Trying to create a questionnaire with invalid data, should return errors', async () => {
+    const invalidObject = {
+      "date": "2022-02-14",
+      "userId": 1,
+      "foodAndDrinks": {
+        "drink": {
+          "water": "lot",
+          "sugaryDrink": "many",
+          "energyDrink": "no",
+          "caffeineDrink": "few",
+          "alcohol": "no"
+        }
+      },
+    }
+    expect(await createQuestionnaire(Questionnaire, invalidObject)).toEqual({
```

Comment:
This call is not mocked, it's using the real mongoose model

Reply:
I have modified the test, check it out later